deploy/examples/cluster-test-multus.yaml

```diff
@@ -0,0 +1,49 @@
+#################################################################################################################
```

Comment:
How about naming the file `cluster-multus-test.yaml`? The test suffix is more consistent with other yaml examples this way.